src/main/kotlin/br/com/creditas/credit_analysis/requests/ContactDto.kt

```diff
@@ -0,0 +1,14 @@
+package br.com.creditas.credit_analysis.requests
+
+import br.com.creditas.credit_analysis.models.PhoneType
+import javax.validation.constraints.Email
+import javax.validation.constraints.Size
+
+data class ContactDto(
+    val type: PhoneType,
+    @Size(min = 11, max = 12)
+    val phoneNumber: String,
+    @Email
+    @Size(max = 255)
+    val email: String
```

Comment:
You could play with some validations here to improve the data, for example, expecting an `@` on the email, nad only allowing digits on the phone number